README.md

```diff
@@ -540,15 +553,58 @@ If you want to have the EnsureExist behavior, see [Resource Update Mode](#Resour
 
 - ie: `kubectl label mtp <your-mtp> deploy.razee.io/debug=true`
 
-### Lock Cluster Updates
+## Cluster Wide Controls
+
+The optional `razeedeploy-config` ConfigMap can be used to customize the
+controller for cluster wide actions.
+
+Because the ConfigMap is optional, if it is created the first time, you must
+restart controller pods, so the deployment can mount the ConfigMap
+as a volume.
+
+Example:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: razeedeploy-config
+  namespace: razeedeploy
+data:
+  lock-cluster: "false"
+  enable-impersonation: "false"
+```
+
+### Lock Cluster
+
+**Key:** `lock-cluster`
+
+**Options:**
+
+- DEFAULT: `false`
+  - Allows the controller to continue normal operations on the cluster.
+- `true`
+  - Prevents the controller from updating resources on the cluster.
+
+### Enable Cluster Wide User Impersonation
+
+**Key:** `enable-impersonation`
+
+**Options:**
 
-Prevents the controller from updating resources on the cluster. If this is the
-first time creating the `razeedeploy-config` ConfigMap, you must delete the running
-controller pods so the deployment can mount the ConfigMap as a volume. If the
-`razeedeploy-config` ConfigMap already exists, just add the pair `lock-cluster: true`.
+- DEFAULT: `false`
+  - Prevents the controller from performing [user impersonation](https://github.com/razee-io/RemoteResource#user-impersonation)
+    in all namespaces. Continues to allow user impersonation in the `razeedeploy`
+    namespace. To prevent privildge escalation, users should be restricted from
+    creating razeedeploy resources.
+- `true`
+  - Allows the controller to perform [user impersonation](https://github.com/razee-io/RemoteResource#user-impersonation)
+    in all namespaces. **See important note below about steps that should be taken
+    to properly configure this feature before enabling.**
```

Comment:
```suggestion
- DEFAULT: `false`
  - Prevents the controller from performing [user impersonation](https://github.com/razee-io/MustacheTemplate#user-impersonation)
    in all namespaces. Continues to allow user impersonation in the `razeedeploy`
    namespace. To prevent privildge escalation, users should be restricted from
    creating razeedeploy resources.
- `true`
  - Allows the controller to perform [user impersonation](https://github.com/razee-io/MustacheTemplate#user-impersonation)
    in all namespaces. **See important note below about steps that should be taken
    to properly configure this feature before enabling.**
```